### ISSUE TITLE
`function-call-argument-newline`: does not work

### ISSUE THREAD

**Issue by jgburet** [eslint-stylistic/eslint-stylistic#307]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Uggly style is not identifed nor fixed.
Other rules are working fine.

`"@stylistic/eslint-plugin": "^1.6.3",`

```typescript
function asdasd(a: number, b: number) {}
asdasd(1,
  2)

function qweqwe(a: number, b: object) {}
qweqwe(1,
    {
        a:1,
        b: 2})

```

```
env:
  browser: true
  es2021: true
  node: true
  jest: true
extends:
  - eslint:recommended
  - plugin:@typescript-eslint/strict-type-checked
  - plugin:@typescript-eslint/stylistic-type-checked
parser: "@typescript-eslint/parser"
parserOptions:
  ecmaVersion: latest
  sourceType: module
  project:
    - "tsconfig.json"
    - "./**/tsconfig.json"
plugins:
  - "@typescript-eslint"
  - "@stylistic"
  - "import"
  - "simple-import-sort"
  - "unicorn"
rules:
  "@stylistic/function-call-argument-newline":
    - warn
    - "consistent"
```

Interestingly, code stays as
```typescript
import { NestFactory } from "@nestjs/core";
import type { NestFastifyApplication } from "@nestjs/platform-fastify";
import { FastifyAdapter } from "@nestjs/platform-fastify";

import { ServiceCoreModule } from "./service-core.module";

async function bootstrap() {
    const app = await NestFactory.create<NestFastifyApplication>(ServiceCoreModule, new FastifyAdapter());
    await app.listen(3_000);
}
```
with "consistent".
However, it is formatted to
```typescript
async function bootstrap() {
    const app = await NestFactory.create<NestFastifyApplication>(ServiceCoreModule,
        new FastifyAdapter());
    await app.listen(3_000);
}
```
with "always" (`@stylistic/max-len` was complaining the line was 108 instead of a 100)

### Reproduction

/

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by jgburet** [comment]
It was conflicting with `function-paren-newline`
